whelk-core/src/main/groovy/whelk/filter/LinkFinder.groovy

```diff
@@ -115,8 +115,9 @@ class LinkFinder {
     }
 
     private void replaceSameAsLinksWithPrimaries(Map data, List path = []) {
-        // If this is a link (an object containing _only_ an id)
+        def exceptedPaths = ['bulk:changeSpec.bulk:deprecate'] as Set
```

Comment:
I think this should be a constant with a more descriptive name and a small comment explaining what it does.